.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: SROS2 CI
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run daily
+    - cron:  '0 12 * * *'
+
+jobs:
+  build_and_test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+    - uses: ros-tooling/setup-ros@0.0.15
+    - uses: ros-tooling/action-ros-ci@0.0.14
+      with:
+        package-name: |
+          sros2
+          sros2_cmake
+    - name: Upload Logs
+      uses: actions/upload-artifact@v1
+      if: failure()
+      with:
+        name: colcon-logs
+        path: ros_ws/log
+
+  # Linter jobs
+  ament_copyright:
+    name: ament_${{ matrix.linter_name }}
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        linter_name: [copyright]
+    steps:
+    - uses: actions/checkout@v1
+    - uses: ros-tooling/setup-ros@0.0.15
+    - uses: ros-tooling/action-ros-lint@0.0.5
+      with:
+        linter: ${{ matrix.linter_name }}
+        package-name: |
+          sros2
+          sros2_cmake
+
+  # action-ros-lint does not support multi work linters like lint_cmake
```

Comment:
Are we confident this will change soon-ish? Or should we remove this section so it doesn't turn into crufty comments?

Reply:
I'd say remove it, I don't personally believe in checking in commented-out code. I don't have a lot of context on this but I did notice that there wasn't a feature request or bug report opened up for it. Please do not hesitate in the future @mikaelarguedas ! I opened this one https://github.com/ros-tooling/action-ros-lint/issues/31

Reply:
Ah, thanks @emersonknapp!

Reply:
> Please do not hesitate in the future @mikaelarguedas ! I opened this one ros-tooling/action-ros-lint#31

I held on as it was a 2 line-fix, I was planning on contributing it rather that opening a feature request ;)
I now had the opportunity to test it so opened the PR is at https://github.com/ros-tooling/action-ros-lint/pull/34